packages/core/src/routes-me/verification-code.ts

```diff
@@ -33,12 +42,35 @@ export default function verificationCodeRoutes<T extends AuthedMeRouter>(
   router.post(
     '/verification-codes/verify',
     koaGuard({
-      body: object({ email: string().regex(emailRegEx), verificationCode: string().min(1) }),
+      body: object({
+        email: string().regex(emailRegEx),
+        verificationCode: string().min(1),
+        action: union([literal('changeEmail'), literal('changePassword')]),
+      }),
```

Comment:
1. we should write response guard for new APIs
2. do we have integration tests covered for new APIs?

Reply:
updated

Reply:
And we don't have integration tests now for these new `me` APIs. Will add them later.
https://linear.app/silverhand/issue/LOG-5600/improve-error-handling-in-ac-profile-page